Src/LinearSolvers/MLMG/AMReX_MLEBTensorOp.cpp

```diff
@@ -479,21 +479,30 @@ MLEBTensorOp::compCrossTerms(int amrlev, int mglev, MultiFab const& mf) const
 	  AMREX_D_TERM(Array4<Real const> const kapxfab = kapmf[0].const_array(mfi);,
 		       Array4<Real const> const kapyfab = kapmf[1].const_array(mfi);,
 		       Array4<Real const> const kapzfab = kapmf[2].const_array(mfi););
-	  
+
+          const auto & bdcv = bcondloc.bndryConds(mfi);
+          Array2D<BoundCond,0,2*AMREX_SPACEDIM-1,0,AMREX_SPACEDIM-1> bct;
+          for (OrientationIter face; face; ++face) {
+              Orientation ori = face();
+              for (int icomp = 0; icomp < AMREX_SPACEDIM; ++icomp) {
```

Comment:
It will be slightly more efficient to do
```
for (int icomp ...)
    for (OrientationIter ...)
```
Could you switch the looping order here at similar other places?

Reply:
Thanks for the review!! I did the changes as suggested.